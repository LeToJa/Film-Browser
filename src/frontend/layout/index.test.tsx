import { describe, it, expect, vi } from 'vitest'
import { render } from '@testing-library/react'
import Layout from '.'

vi.mock('react-router-dom', () => ({
  Link: ({ children }: { children: string }) => <a>{children}</a>,
}))

describe('Layout', () => {
  it('matches snapshot with children', () => {
    const { container } = render(
      <Layout>
        <div>Child Content</div>
      </Layout>
    )
    expect(container).toMatchSnapshot()
  })
})
