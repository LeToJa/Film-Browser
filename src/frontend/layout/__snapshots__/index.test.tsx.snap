// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Layout > matches snapshot with children 1`] = `
<div>
  <div
    id="app-header"
  >
    <nav
      class="general-body"
    >
      <a>
        Home
      </a>
      <a>
        Wishlist
      </a>
    </nav>
  </div>
  <section
    class="general-body"
  >
    <div>
      Child Content
    </div>
  </section>
</div>
`;
